### PR TITLE
DIY chapter added.

### PR DESCRIPTION
**Pull request by asendecka**
A first draft of the DIY chapter added.

### CODE REVIEW

**Comment by patjouk** [comment]
This is awesome :heart_eyes_cat: 
Tell me when you're reading to merge :wink:

File: diy/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# DIY: how to make a workshop beautiful on budget
+
+We strive to make each Django Girls event one of the kind experience. If you look at the pictures from the past workshops you'll notice amazing decorations, photo booths, flowers, cupcakes and many other exciting things. This one of the things that makes Django Girls unique, fun and exciting and we love it.
```
Comment (phalt) [inline diy/README.md]: _"make each Django Girls event **a** one of the kind experience"_ :)

File: diy/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+This chapter will give you some tips and examples, how - with a little of DIY and good ideas and hacks - you could create something magical with very little money. 
+
+## Decorations
+
+There are different ways to make the workshop look amazing. If you are a crafty type, preparing decorations would be very easy (but might take some time). If you hate this kind of stuff, we have some tips for you, too.
+
+### Balloons
+
+* many of the Django Girls events order big, [gold letter balloons](https://www.flickr.com/photos/djangogirls/15624332819/in/album-72157649335413325/). In on-line shops or on eBay you could find them for 2-6 dollars each (depending on shop and size of the balloon). Order in advance so it will come in time for your event. If you have smaller budget, instead of having whole “Django Girls” (or any other longer text), you could go for something shorter, like: "YAY".
```
Comment (phalt) [inline diy/README.md]: Caps on the first word :)

Comment (phalt) [inline diy/README.md]: Hehe in a few places actually. Sorry! :yum:

**Comment by patjouk** [comment]
Thanks @asendecka :heart:

**Comment by asendecka** [comment]
And thanks to @agagata and @phalt for feedback on this one :). :sparkle: :heart: :santa:

**Pull request merged**